README-template.md

```diff
@@ -1,5 +1,7 @@
 [![Build Status](https://travis-ci.com/IBM/[MODEL REPO NAME].svg?branch=master)](https://travis-ci.com/IBM/[MODEL REPO NAME]) [![Website Status](https://img.shields.io/website/http/[MODEL DOCKER TAG].max.us-south.containers.appdomain.cloud/swagger.json.svg?label=api+demo)](http://[MODEL DOCKER TAG].max.us-south.containers.appdomain.cloud/)
 
+[<img src="docs/deploy-max-to-ibm-cloud-with-kubernetes-button.png" width="400px">](http://ibm.biz/max-to-ibm-cloud-tutorial)
```

Comment:
Are you sure the `src` link would work? It'll probably leads to a GitHub page rather than a raw png file.

Reply:
The file that the `src` refers to is also being added in this PR, but doesn't render here. You can go to my branch to see what it will look like [here](https://github.com/splovyt/MAX-Skeleton/blob/readme_update/README-template.md)

Reply:
Looks awesome!

Reply:
Thanks :)))))))))